sonar.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: "org.sonarqube"
+
+def ignoredModules = [
+        'stream-chat-android-ai-assistant',
+        'stream-chat-android-previewdata',
+        'stream-chat-android-docs',
+        'stream-chat-android-benchmark',
+        'stream-chat-android-compose-sample',
+        'stream-chat-android-e2e-test',
+        'stream-chat-android-test',
+        'stream-chat-android-ui-guides',
+        'stream-chat-android-ui-components-sample',
+        'stream-chat-android-pushprovider-firebase',
```

Comment:
This module doesn't exist anymore

Reply:
All 4 modules? So it's safe to delete these lines, isn't it?